### DISCUSSION TITLE
Move <SubMenu> and <Menu.Item> by drag & drop

### DISCUSSION THREAD

**Discussion by brightestbulb** [ant-design/ant-design#31429] [Ideas]
Hello.

I am using the menu of antd design well. (https://ant.design/components/menu/)

Do you have any plans to develop the ability to move **`SubMenu`** and **`Menu.Item`** to another **`SubMenu`** by drag & drop?

Having this feature would be a great help to many people.

Thank you always.

**Comment by imkitukov** [comment]
Any update on this? 

Seems like on antd v5,  Menu Item ignores onDragStart and now dragging and reorder Menu items is not possible.

It was working fine with v3. Anyone faced the same issue with version 5?
